externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/operation/PlanOnlyStatement.scala

```diff
@@ -36,6 +36,11 @@ class PlanOnlyStatement(
   extends SparkOperation(OperationType.EXECUTE_STATEMENT, session) {
 
   private val operationLog: OperationLog = OperationLog.createOperationLog(session, getHandle)
+  private val configExcludes: Option[String] =
+    spark.conf.getOption(OPERATION_PLAN_ONLY_EXCLUDES.key)
+  private val planExcludes: Seq[String] =
+    if (configExcludes.isEmpty) session.sessionManager.getConf.get(OPERATION_PLAN_ONLY_EXCLUDES)
+    else configExcludes.get.split(",").toSeq
```

Comment:
the priority of session conf should be higher than spark conf

Reply:
@ulysses-you, IMO, the priority of the spark conf is higher than session conf. cc @yaooqinn

Reply:
+1 for @ulysses-you 's suggestion.

Reply:
SessionManger's getConf is not session conf, its engine conf

Reply:
@ulysses-you, as you mentioned, the [SparkOperation.scala#L73](https://github.com/apache/incubator-kyuubi/blob/master/externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/operation/SparkOperation.scala#L73) has the same problem, right?

Reply:
I see, we have merged session conf into spark session conf. LGTM